tests/_test_msui/test_mscolab.py

```diff
@@ -51,6 +52,10 @@
 
 class Test_Mscolab_connect_window():
     def setup_method(self):
+        keyring.get_keyring().reset()
+        import mslib.mscolab.conf
+        mslib.mscolab.conf.mscolab_settings.enable_basic_http_authentication = True
+
```

Comment:
we maybe should move these tests, which needs a compilation to a special tests_run_once on same level as tests/ and have a pytest script action which collects the tests below and runs them one after the other in an individual pytest process.

Reply:
the basic_http_auth has no effect and should have been removed.